src/query/src/sql.rs

```diff
@@ -400,6 +400,21 @@ pub async fn show_index(
         query_ctx.current_schema()
     };
 
+    let fulltext_index_expr = case(col("constraint_name").like(lit("%FULLTEXT INDEX%")))
+        .when(lit(true), lit("greptime-fulltext-index-v1"))
+        .otherwise(null())
+        .context(error::PlanSqlSnafu)?;
+
+    let inverted_index_expr = case(
+        col("constraint_name")
+            .like(lit("%INVERTED INDEX%"))
+            .or(col("constraint_name").like(lit("%PRIMARY%")))
+            .or(col("constraint_name").like(lit("%TIME INDEX%"))),
```

Comment:
The time index doesn't need to have `greptime-inverted-index-v1` type. It can be null.

Reply:
updated